test/system/030-run.bats

```diff
@@ -818,8 +818,19 @@ EOF
 
 @test "podman run defaultenv" {
     run_podman run --rm $IMAGE printenv
-    assert "$output" =~ "TERM=xterm" "env includes TERM"
+    assert "$output" !~ "TERM=" "env doesn't include TERM by default"
     assert "$output" =~ "container=podman" "env includes container=podman"
+    
+
+    run_podman run -t=false --rm $IMAGE printenv
+    assert "$output" !~ "TERM=" "env doesn't include TERM"
```

Comment:
```suggestion
    run_podman 1 run -t=false --rm $IMAGE printenv TERM
    assert "$output" == "" "env doesn't include TERM"
```

printenv will exit 1 when the env is not found so we can juts use that here as well as we can check the exit code with run_podman <NUM>